sky/utils/kubernetes/sshjump-lcm.py

```diff
@@ -0,0 +1,93 @@
+"""Manages lifecycle of sshjump pod.
+
+This script runs inside sshjump pod as the main process (PID 1).
+"""
+import datetime
+import os
+import pytz
+import sys
+import time
+
+from kubernetes import client, config
+
+# Load kube config
+config.load_incluster_config()
+
+v1 = client.CoreV1Api()
+
+current_name = os.getenv("MY_POD_NAME")
+current_namespace = os.getenv("MY_POD_NAMESPACE")
+
+# The amount of time in seconds where no Ray pods exist in which after that time
+# sshjump pod terminates itself
+alert_threshold = int(os.getenv("ALERT_THRESHOLD", "300"))

```

Comment:
```suggestion
alert_threshold = int(os.getenv("ALERT_THRESHOLD", "600"))
```
Was  wondering if this was meant to be `600` just like from `kubernetes-sshjump.yml.j2`?

Reply:
You are right. Thanks.